packages/ui/src/components/va-modal/VaModal.vue

```diff
@@ -308,10 +299,16 @@ export default defineComponent({
     z-index: var(--va-modal-overlay-z-index);
     width: var(--va-modal-overlay-width);
     height: var(--va-modal-overlay-height);
+  }
 
-    &--transition {
-      @include va-modal-transition(true);
```

Comment:
We can remove this old mixin from resources now 👍

Reply:
I think I removed that in the subsequent commit. I think I found it right after the request for review. I am sorry that I ask hastily before recheck anything left to do. I am prone to making that kind of mistake, just like washing disk and think I am finish and clean the dishcloth only to find there is one left undone in the hind of eye. I think I have to make it a routine check here.